data-structures/linked_list/mergeList.py

```diff
@@ -0,0 +1,33 @@
+# from linked_list import LinkedList
+
+
+def ll_merge(arg2, arg1):
+    if arg2._len <= 0 and arg1._len <= 0:
+        return False
+
+    if arg2.head is None:
+        return arg1
+    if arg1.head is None:
+        return arg2
+
+    if True:
+        ptr_left, ptr_right = arg1.head, arg2.head
+        while ptr_right is not None:
+            # if ptr_left._next:
+            arg1.insert_before(ptr_left.val, ptr_right)
```

Comment:
This only works if all the values in your linked list are unique. Also, you are inserting nodes as the value for the new node, so you might want to change `ptr_right` to `ptr_right.val`.